extensions/scaleway/src/iam/queries/useUserQuery.ts

```diff
@@ -0,0 +1,15 @@
+import type { IAM } from '@scaleway/sdk'
+import { useDataLoader } from '@scaleway/use-dataloader'
+import { useAPI } from '../../providers'
+
+type DataLoaderOption<T> = Parameters<typeof useDataLoader<T>>[2]
+
+export const useUserQuery = (
+  params: IAM.v1alpha1.GetUserRequest,
+  dataloaderOptions?: DataLoaderOption<IAM.v1alpha1.User>
+) => {
+  const { iamV1Alpha1 } = useAPI()
+  const key = ['iam', 'user', Object.entries(params).sort()].flat(3)
```

Comment:
From above code (applications, policies, groups) I assume this is plural 

```suggestion
  const key = ['iam', 'users', Object.entries(params).sort()].flat(3)
```